### DISCUSSION TITLE
julia prints "Inf" instead of ZERO

### DISCUSSION THREAD

**Discussion by KevinMan46** [JuliaLang/julia#52947] [General]
Did someone else noted that the result of the following code shows "Inf" instead of zero?

```
infOrzero = 1 / 2^3000
print(infOrzero)

#Shows Inf, not 0.0
```
In other languages the result is zero......., it seem only happens with 2 based pow.

**Comment by oscardssmith** [comment]
You want 2.0^3000. `Int^`Int` overflows here so you have `1/0`.

**Reply by KevinMan46** [reply]
Thanks, I'm learning recently the language, thanks for your answer.

**Reply by oscardssmith** [reply]
no problem. This is a fairly common tripping point from users who are coming from R or Python (or other languages that don't expose machine width integers)
